### ISSUE TITLE
[Feature] Example sentences for each definition

### ISSUE THREAD

**Issue by DataKinds** [jProgr/TokiPonaDictionary#6]
toki!

mi li jan toki sin pi toki pona. 

It'd be helpful to me in the learning process to be able to see example sentences associated with each word to see how it can be used in context. Some of the more interesting grammar can be made much clearer using examples -- "la", for example, does not have a clear function at all using the dictionary reference, despite its couple roles in speech being quite well defined.

I'd be happy to contribute some of these if they were proofread afterwards.

jan Tala

**Comment by jProgr** [comment]
Good idea. I will try to add one or two simple sentences to each word. Your contributions will be gladly received, just keep in mind that any contribution will be in the public domain due to the license of this project https://github.com/jProgr/TokiPonaDictionary/blob/master/LICENSE.md

**Comment by jProgr** [comment]
I will use this comment to track and collect the sentences.

Words and its sentences (if done):
- [ ] a
- [ ] akesi
- [ ] ala
- [ ] alasa
- [ ] ali
- [ ] anpa
- [ ] ante
- [ ] anu
- [ ] awen
- [ ] e
- [ ] en
- [ ] esun
- [ ] ijo
- [ ] ike
- [ ] ilo
- [ ] insa
- [ ] jaki
- [ ] jan
- [ ] jelo
- [ ] jo
- [ ] kala
- [ ] kalama
- [ ] kama
- [ ] kasi
- [ ] ken
- [ ] kepeken
- [ ] kili
- [ ] kin
- [ ] kipisi
- [ ] kiwen
- [ ] ko
- [ ] kon
- [ ] kule
- [ ] kulupu
- [ ] kute
- [ ] la
- [ ] lape
- [ ] laso
- [ ] lawa
- [ ] len
- [ ] lete
- [ ] li
- [ ] lili
- [ ] linja
- [ ] lipu
- [ ] loje
- [ ] lon
- [ ] luka
- [ ] lukin
- [ ] lupa
- [ ] ma
- [ ] mama
- [ ] mani
- [ ] meli
- [ ] mi
- [ ] mije
- [ ] moku
- [ ] moli
- [ ] monsi
- [ ] monsuta
- [ ] mu
- [ ] mun
- [ ] musi
- [ ] mute
- [ ] namako
- [ ] nanpa
- [ ] nasa
- [ ] nasin
- [ ] nena
- [ ] ni
- [ ] nimi
- [ ] noka
- [ ] o
- [ ] oko
- [ ] olin
- [ ] ona
- [ ] open
- [ ] pakala
- [ ] pali
- [ ] palisa
- [ ] pan
- [ ] pana
- [ ] pi
- [ ] pilin
- [ ] pimeja
- [ ] pini
- [ ] pipi
- [ ] poka
- [ ] poki
- [ ] pona
- [ ] pu
- [ ] sama
- [ ] seli
- [ ] selo
- [ ] seme
- [ ] sewi
- [ ] sijelo
- [ ] sike
- [ ] sin
- [ ] sina
- [ ] sinpin
- [ ] sitelen
- [ ] sona
- [ ] soweli
- [ ] suli
- [ ] suno
- [ ] supa
- [ ] suwi
- [ ] tan
- [ ] taso
- [ ] tawa
- [ ] telo
- [ ] tenpo
- [ ] toki
- [ ] tomo
- [ ] tu
- [ ] unpa
- [ ] uta
- [ ] utala
- [ ] walo
- [ ] wan
- [ ] waso
- [ ] wawa
- [ ] weka
- [ ] wile

**Comment by jProgr** [comment]
Suggestions:
- a: ike a!. Oh my!
- akesi: akesi ni li kala e soweli lili. That reptile is eating a rat.
- ala: sina wile ala wile moku? wile ala. Do you want to eat? I don't want to eat.

**Comment by jProgr** [comment]
Suggestions:
- alasa: o alasa e soweli! Hunt the dog!
- ale: ale li pona. Everything will be ok.
- anpa: uta mi li lon anpa nena kon mi. My mouth is under my nose.
- ante: nimi ante pi mi mute. Our words are different.
- anu: sina wile moku e suwi anu kili? Do you want to eat cookies or fruit?

**Comment by TheSeptembre** [comment]
Suggestions : 
* e: sina moku e kili. You eat a fruit. 
* esun: tenpo ni la mi toki esun. I greeted the sun today. 
* jo: sina jo tenpo kama pona! Have a good afternoon!
* sina ken toki e toki pona. You can speak toki pona.
* wawa: sina wawa. You are strong.
